packages/lesswrong/lib/sql/CreateTableQuery.ts

```diff
@@ -0,0 +1,24 @@
+import Query from "./Query";
+import Table from "./Table";
+
+/**
+ * Builds a Postgres query to create a new database table with the schema
+ * defined by `table`. If the table already exists, setting `ifNotExists`
+ * to true will prevent an error from being thrown.
+ */
+class CreateTableQuery<T extends DbObject> extends Query<T> {
+  constructor(table: Table, ifNotExists = false) {
+    const fields = table.getFields();
+    super(table, [
+      `CREATE TABLE${ifNotExists ? " IF NOT EXISTS" : ""}`,
+      table,
+      `(_id ${fields["_id"].toString()} PRIMARY KEY`,
+    ]);
+    for (const field of Object.keys(fields).filter((field) => field !== "_id")) {
```

Comment:
nit: for readability
```suggestion
    for (const field of Object.keys(fields).filter((fieldName) => fieldName !== "_id")) {
```
also isn't there only ever one `_id` field, why can't we just `find` that field?

Reply:
That's a `!==`, not a `===`